homeassistant/components/climate/ecobee.py

```diff
@@ -22,16 +22,23 @@
 _LOGGER = logging.getLogger(__name__)
 
 ATTR_FAN_MIN_ON_TIME = 'fan_min_on_time'
+ATTR_RESUME_ALL = 'resume_all'
 
 DEPENDENCIES = ['ecobee']
 
 SERVICE_SET_FAN_MIN_ON_TIME = 'ecobee_set_fan_min_on_time'
+SERVICE_RESUME_PROGRAM = 'ecobee_resume_program'
 
 SET_FAN_MIN_ON_TIME_SCHEMA = vol.Schema({
     vol.Optional(ATTR_ENTITY_ID): cv.entity_ids,
     vol.Required(ATTR_FAN_MIN_ON_TIME): vol.Coerce(int),
 })
 
+RESUME_PROGRAM_SCHEMA = vol.Schema({
+    vol.Optional(ATTR_ENTITY_ID): cv.entity_ids,
+    vol.Optional(ATTR_RESUME_ALL): cv.boolean,
```

Comment:
Not necessarily.  The service does control the state of the thermostat.  My comment above is only that the `refresh_all` parameter isn't a representation of the new state like the parameter for `set_away_mode` is.  `refresh_all` is only the behavior of the service, not the new state itself.  But you know the design principles of the project much better than I do, so I'll yield to your final verdict on the matter.  Thanks!